datafusion/results/single.json

```diff
@@ -1,58 +1,58 @@
 {
     "system": "DataFusion (single)",
-    "date": "2023-02-21",
+    "date": "2023-04-11",
     "machine": "c6a.4xlarge, 500gb gp2",
     "cluster_size": 1,
-    "comment": "",
+    "comment": "v22.0.0 (34c9bce)",
 
     "tags": ["Rust", "column-oriented", "embedded", "stateless"],
 
     "load_time": 0,
     "data_size": 14779976446,
 
     "result": [
-[2.656, 0.227, 0.224],
-[0.097, 0.080, 0.076],
-[0.160, 0.139, 0.142],
-[0.336, 0.123, 0.124],
-[2.652, 2.677, 2.663],
-[1.749, 1.818, 1.701],
-[0.103, 0.084, 0.084],
-[0.102, 0.078, 0.093],
-[9.134, 2.867, 2.871],
-[61.637, 62.583, 65.014],
-[0.492, 0.382, 0.389],
-[0.503, 0.416, 0.414],
-[2.071, 2.968, 2.008],
-[4.314, 3.640, 3.624],
-[2.147, 2.081, 2.077],
-[3.546, 3.425, 6.533],
-[5.092, 5.098, 4.989],
-[4.410, 4.315, 4.352],
-[14.138, 10.743, 11.860],
-[0.206, 0.109, 0.108],
-[10.058, 1.554, 1.520],
-[11.167, 1.846, 1.825],
-[22.119, 4.371, 4.366],
-[56.076, 11.756, 11.948],
-[2.559, 0.590, 0.583],
-[0.746, 0.489, 0.491],
-[2.557, 0.606, 0.590],
-[9.576, 2.412, 2.487],
-[8.925, 5.805, 5.744],
-[0.606, 0.580, 0.557],
-[3.173, 2.885, 2.881],
-[7.420, 5.220, 7.927],
+[2.646, 0.225, 0.226],
+[0.099, 0.079, 0.079],
+[0.173, 0.138, 0.139],
+[0.349, 0.126, 0.125],
+[2.790, 6.718, 2.717],
+[1.911, 1.753, 1.770],
+[0.090, 0.079, 0.076],
+[0.106, 0.081, 0.081],
+[3.032, 3.054, 3.675],
+[3.634, 3.564, 3.647],
```

Comment:
It was a performance regression since https://github.com/ClickHouse/ClickBench/pull/77/files#r1109213240
I think it is fixed by https://github.com/apache/arrow-datafusion/issues/5325